Mathlib/Order/Minimal.lean

```diff
@@ -315,7 +313,7 @@ theorem IsAntichain.maximals_lowerClosure (hs : IsAntichain (· ≤ ·) s) :
 
 section Image
 
-variable {f : α → β} {r : α → α → Prop} {s : β → β → Prop}
+variable {β : Type*} {f : α → β} {r : α → α → Prop} {s : β → β → Prop}
```

Comment:
Maybe hoist `beta` to the top of the file, if that works?

Reply:
It does; done!